server/sonar-web/src/main/js/apps/projects/components/PerspectiveSelectOption.js

```diff
@@ -31,21 +32,22 @@ type Props = {
   onFocus: (Option, MouseEvent) => void,
   onSelect: (Option, MouseEvent) => void
 };
+*/
 
 export default class PerspectiveSelectOption extends React.PureComponent {
-  props: Props;
+  /*:: props: Props; */
 
-  handleMouseDown = (event: MouseEvent) => {
+  handleMouseDown = (event /*: MouseEvent */) => {
     event.preventDefault();
     event.stopPropagation();
     this.props.onSelect(this.props.option, event);
   };
 
-  handleMouseEnter = (event: MouseEvent) => {
+  handleMouseEnter = (event /*: MouseEvent */) => {
     this.props.onFocus(this.props.option, event);
   };
 
-  handleMouseMove = (event: MouseEvent) => {
+  handleMouseMove = (event /*: MouseEvent */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)